# Skill definition
string name
string[] initial_preconditions
State[] intermediate_states
string[] final_postconditions